### PR TITLE
Copy latest docs changes

### PR DESCRIPTION
**Pull request by jason-seqera**
I think we want to copy these files back into this repo as the ongoing source of truth for this.

@llewellyn-sl @justinegeffen

### CODE REVIEW

**Comment by jason-seqera** [comment]
@munishchouhan, this lifts and shifts the latest version of the Wave documentation back into this repository; for expediency we had completed this work in the new `docs` repo, with the goal of transferring it back here after.

Thanks!

File: docs/api.md [hunk 1]
```diff
@@ -1,109 +1,103 @@
...
-    - navigation
-    - footer
+sidebar_position: 12
 ---
 
-# API reference
+This page summarizes the API provided by the Wave container service.
 
-This page summarizes the API provided by the Wave service.
+## `/container-token`
```
Comment (munishchouhan) [inline docs/api.md:8]: shouldn't we provide the HTTP method for this endpoint?
in this case its POST

**Comment by munishchouhan** [comment]
@pditommaso @jason-seqera we do have `GET /container-token/{token}'` API endpoint also.

**Comment by jason-seqera** [comment]
Added link fixes from: https://github.com/seqeralabs/docs/pull/18

**Comment by pditommaso** [comment]
@jason-seqera are you find to merge this?

**Comment by jason-seqera** [comment]
@pditommaso, I'm working on getting it to work with the new docs previewer first: https://github.com/seqeralabs/docs-previewer

File: docs/api.md (hunk 1 above)
Comment (jason-seqera) [reply docs/api.md:7]: @munishchouhan, I can create a new PR specifically to update the API methods.

**Comment by jason-seqera** [comment]
@pditommaso, this works with the docs previewer container now. I think it's ready to merge and then we can iterate on the Wave docs in this repo.

**Comment by pditommaso** [comment]
Excellent. Can we consider the docs in this repo the source of truth for future changes?

**Comment by jason-seqera** [comment]
@pditommaso, yes, we're continuing that model w/ the docs directory as the source of truth. Thanks!

**Pull request merged**